dasharo-security/secure-boot.robot

```diff
@@ -233,6 +233,108 @@ SBO008.001 Attempt to enroll the key in the incorrect format (firmware)
     Select File In File Explorer    DB.txt
     Read From Terminal Until    ERROR: Unsupported file type!
 
+SBO009.001 Check support for rsa2k signed certificates
+    Download ISO And Mount As USB    ${RSA2_K_TEST_NAME}    ${RSA2_K_TEST_URL}
```

Comment:
@tym2k1 right now I got this for SBO009.001

```
λ robot -L TRACE -v rte_ip:127.0.0.1 -v snipeit:no -v config:qemu -t "SBO009.001*" dasharo-security/secure-boot.robot
==============================================================================
Secure-Boot                                                                   
==============================================================================
SBO009.001 Check support for rsa2k signed certificates                | FAIL |
Keyword 'usb-hid-msc-lib.Download ISO And Mount As USB' expected 3 arguments, got 2.
------------------------------------------------------------------------------
```

This keyword is missing SHA as an argument. Also, can we use similar syntax to this one https://github.com/Dasharo/open-source-firmware-validation/blob/uefi-sb-release/dasharo-security/secure-boot.robot#L190 - can we download this image to `DL_CACHE_DIR` ?

Reply:
Addressed and squashed. The `robotidy` hook of pre-commit forced multiline syntax for `ecdsa256`, `ecdsa384` and `ecdsa521`